invenio_oauthclient/views/client.py

```diff
@@ -271,3 +271,19 @@ def rest_disconnect(remote_app):
         return _disconnect(remote_app)
     except OAuthRemoteNotFound:
         abort(404)
+
+
+@blueprint.route("/logout")
```

Comment:
If we decide to got for this I think we should split between `logout_url´ and `rest_logout_url` because I guess while using the rest you would like to redirect to rest endpoint as well ?

Reply:
lets do as an improvement and ship this one